src/content/docs/distribute/flatpak.mdx

```diff
@@ -105,10 +128,32 @@ However, this is not recommended for the final build of the flatpak, as your loc
 This can introduce various bugs that can be very hard to find.
 Therefore, it is recommended to build the library your program depends on from source inside the flatpak as a build step.
 
-You can read more about this [in the flatpak documentation](https://docs.flatpak.org/en/latest/dependencies.html#bundling)
+## Submitting to flathub
+***1. Fork The [Flathub Repository]("https://github.com/flathub/flathub/fork")***
```

Comment:
```suggestion
***1. Fork The [Flathub Repository](https://github.com/flathub/flathub/fork)***
```
(it links to https://deploy-preview-2279--tauri-v2.netlify.app/distribute/flatpak/%22https://github.com/flathub/flathub/fork%22 right now)